cmd/kubeadm/app/constants/constants.go

```diff
@@ -398,6 +398,17 @@ const (
 
 	// CgroupDriverSystemd holds the systemd driver type
 	CgroupDriverSystemd = "systemd"
+
+	// The username of the user that kube-controller-manager runs as if RootlessControlPlane feature-gate is enabled.
```

Comment:
```suggestion
	// The username of the user that kube-controller-manager runs.
```

suggesting to exclude the FG mention in the comment, so that we don't have to update once the FG is GA and is removed. FGs are ephemeral.

Reply:
Done. Good catch!